### ISSUE TITLE
[Fixed in GH-6052] Safari Firefox: Missing Visual Focus on My Account: Add New/Edit Address

### ISSUE THREAD

**Issue by developpeurweb** [SAP/cloud-commerce-spartacus-storefront#4331]
![Screen Shot 2019-08-29 at 8 50 26 PM](https://user-images.githubusercontent.com/15113219/63985703-bdd4e380-ca9e-11e9-8d8e-cc771bdcd6cd.png)

**Comment by kacperknapik** [comment]
- Chrome: ✔ (note: submit button not active when form empty - done in a11y epic)
- Firefox: ✔ (note: focus border is huge)
- Safari: ❓

**Comment by KateChuen** [comment]
Chrome: OK

Missing for:
- edit & delete buttons on Firefox
- Add new address on safari
- Back to address list & Update Address & Add Address buttons on Safari
- Set as default checkbox on Safari

**Comment by developpeurweb** [comment]
**For QA:**

- To test in Safari please refer to [these instructions](https://github.com/SAP/cloud-commerce-spartacus-storefront/issues/5041#issuecomment-558883497).
- This got fixed in ticket GH-6052
- Theme `calydon` should be enabled to see changes.
- Visible focus outline overlapping adjacent ui elements may happen and may not be an issue, if in doubt please open a ticket per element or reach out to UI developer.
- Please separate visible focus issues (CSS: element without outline) from tabbing issues (HTML/TS: element not reachable with keyboard).

![Screen Shot 2020-01-30 at 3 49 23 PM](https://user-images.githubusercontent.com/15113219/73489341-723da300-4378-11ea-9120-69942899361a.png)
![Screen Shot 2020-01-30 at 3 50 23 PM](https://user-images.githubusercontent.com/15113219/73489342-723da300-4378-11ea-9c90-92040dcb3cbe.png)
![Screen Shot 2020-01-30 at 3 50 39 PM](https://user-images.githubusercontent.com/15113219/73489344-723da300-4378-11ea-9eae-7b2a2822dc96.png)
![Screen Shot 2020-01-30 at 3 50 31 PM](https://user-images.githubusercontent.com/15113219/73489345-72d63980-4378-11ea-836d-61e04fa82af7.png)
![Screen Shot 2020-01-30 at 3 51 16 PM](https://user-images.githubusercontent.com/15113219/73489343-723da300-4378-11ea-874d-e9679b19721e.png)

**Comment by KateChuen** [comment]
Tested on dev3.
- Chrome ✅
- Firefox ✅
- Safari ✅

Only issue is pressing ENTER on the _Edit_ button does not redirect user to edit address form page.
